program/rust/src/tests/test_upd_price_v2.rs

```diff
@@ -141,11 +139,10 @@ fn test_upd_price_v2() -> Result<(), Box<dyn std::error::Error>> {
         assert_eq!(price_data.comp_[0].latest_.status_, PC_STATUS_TRADING);
         assert_eq!(price_data.valid_slot_, 1);
         assert_eq!(price_data.agg_.pub_slot_, 3);
-        assert_eq!(price_data.agg_.price_, 42);
+        assert_eq!(price_data.agg_.price_, 81);
         assert_eq!(price_data.agg_.conf_, 2);
         assert_eq!(price_data.agg_.status_, PC_STATUS_TRADING);
-
-        assert_eq!(price_data.price_cumulative.price, 3 * 42);
+        assert_eq!(price_data.price_cumulative.price, 42 + 2 * 81);
```

Comment:
this doesn't make sense. I think the right answer is `42 * 2 + 81` (because it was 42 on slot 1, 42 on slot 2, and then 81 on slot 3)

Reply:
hm... it looks like this was the behavior of price cumulative before this PR though. That doesn't make a lot of sense to me though.

Reply:
yeah this was the existing behavior, we can change this if we want to but imo we should do this on a separate PR